src/pages/store/cart.tsx

```diff
@@ -254,6 +254,13 @@ const getServerSidePropsFunc: GetServerSideProps = async ({ req, res }) => {
       .filter(
         event => !(event.orders && event.orderLimit && event.orders.length > event.orderLimit)
       )
+      // filter out events that have a start time less than 2 days from now
+      .filter(event => {
+        const startTime = new Date(event.start);
+        const now = new Date();
+        now.setDate(now.getDate() + 2);
```

Comment:
Doing it this way might make it prone to edge cases when days are 23 or 25 hours long. I think it's fine, but it might not match the backend perfectly

I think a more precise implementation would compare the difference in milliseconds with 2 days' worth of milliseconds, rather than relying on how the date is represented in the user's time zone

Reply:
https://stackoverflow.com/questions/12795767/trying-to-add-3-days-in-milliseconds-to-current-date

maybe

Reply:
yeah, could do something like `startTime.getTime() >= now.getTime() + PICKUP_ADVANCE_TIME` or something, where `PICKUP_ADVANCE_TIME = 1000 * 60 * 60 * 24 * 2`

Reply:
See updated code, does that have the same effect? Lots of different Date methods and ways of representing them